compiler/src/dotty/tools/dotc/core/NullOpsDecorator.scala

```diff
@@ -8,16 +8,35 @@ import dotty.tools.dotc.core.Types.{AndType, ClassInfo, ConstantType, OrType, Ty
 object NullOpsDecorator {
 
   implicit class NullOps(val self: Type) {
+    def findAndStripNull(implicit ctx: Context): (Type, Boolean, Boolean) = {
```

Comment:
I thought we're changing this back to `normNullableUnion` and returning just a `Type`?

Reply:
I am still working on this PR